internal/website/content/tutorials/order.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Order"
+date: 2019-07-09T10:22:39-04:00
+draft: true
+showInSidenav: false  # only for sections (any level)
+pagesInSidenav: false  # only for top-level sections
+weight: 5
+toc: true
+---
+
+In this tutorial, we will run a Go CDK application called Order
+on a local machine.
+
+<!--more-->
+
+Order is a sample application that lets users place orders to convert images to
+PNG format, and to view the results. The main business logic is written in a
+cloud-agnostic manner using the generic APIs for blob, pubsub and docstore.
+
+The Order application has two parts: a web frontend, and an image-processing
+backend called a processor. They communicate over a pubsub topic, store order
+information in a docstore collection, and store image files in a blob bucket.
+
+## Prerequisites
+
+You will need to install the following software for this tutorial:
+
+-   [Git](https://git-scm.com/)
+-   [Go](https://golang.org/doc/install)
+
+Then you need to clone the Go CDK repository. The
+repository contains the Order sample.
+
+```shell
+git clone https://github.com/google/go-cloud.git
+cd go-cloud/samples/order
+```
+
+## Building
+
+Run the following in the `samples/order` directory:
+
+```shell
+go build
+```
+
+## Running Locally
+
+If you run `order` with no arguments, both the frontend and the processor will
+run together in the same process. 
+
+```shell
+./order
+```
+The frontend is now running on http://localhost:10538.
+
+Visit the home page in your browser and click "Convert an Image".
+
+Enter an email address (it need not be real) and select any image file from your
+computer. Then click Submit.
+
+Now visit the order list page by returning to the home page and clicking "List
+Conversions". It make take a few seconds to process the order (thanks to an
+artificial delay in the processor), so refresh the page until you see your order
+in the list.
```

Comment:
Should this also say something about the changes needed to run on real cloud?

Reply:
Said a little.